.github/workflows/tests.yml

```diff
@@ -39,3 +39,72 @@ jobs:
         npm rebuild node-sass
     - name: Run tests
       run: yarn test
+
+  visual_tests:
+    name: Frontend Visual Tests
+    needs: test
+    if: ${{ github.event_name == 'pull_request' && needs.test.result == 'success' }}
+    runs-on: ubuntu-latest
+    outputs:
+      percy_url: ${{ steps.extract-url.outputs.percy_url }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+      - name: Cache Node.js modules
+        uses: actions/cache@v4
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+      - name: Install dependencies
+        run: |
+          yarn --frozen-lockfile
+          npm rebuild node-sass
+      - name: Start server
+        run: |
+          yarn dev &
+          sleep 60 &&
+          curl -sSf http://localhost:4000/testing-playground
+        env:
+          SHELL: /bin/bash
+      - name: Run visual tests
+        id: run-visual-tests
```

Comment:
Do we need these ids on these steps?

Reply:
Ahh I was trying different ways I could capture the step outputs. The `run-visual-tests id` is not needed, but the `extract-url id` is being used to refer to the extracted url output so that it can be accessed later. I'll remove the run-visual-tests id from there.